schemas/service-user.schema.json

```diff
@@ -10,6 +10,7 @@
 		"salutation",
 		"firstName",
 		"lastName",
+		"anonymised",
```

Comment:
I think this change will need to be done in 2 parts - this is what we have had to do previously.
1 - add the new field (lines 54-58), but do not make it required (no line 13).
2 - then applications that use this (eg CBOS) can be updated to populate this field.
3 - when that has happened, then it can be added to "required" list.

If we don't do it this way, CBOS will think that our payloads (which do no include "anonymised" are invalid, and will NOT broadcast any service messages.